distribution/docs/src/main/resources/content/_endpoints/csw-endpoint.adoc

```diff
@@ -1804,7 +1804,49 @@ The `RecordProperty` contains a `Name` and `Value` element.
 The `Name` element is used to specify the name of the record property to be updated.
 The `Value` element contains the value that will be used to update the record in the catalog.
 The values in the `Update` will completely replace those that are already in the record.
-A property is removed only if the `RecordProperty` contains a `Name` but not a `Value`.
+
+Some properties are given default `Value`s if no `Value` is provided.
+
+.`RecordProperty` Default Values
+[cols="1m,1" options="header"]
+|===
+
+|Property
+|Default Value
+
+|metadata-content-type
+|Resource
+
+|created
+|_current time_
+
+|modified
+|_current time_
+
+|effective
+|_current time_
+
+|metadata-content-type-version
+|DIB20
```

Comment:
❗️we should have a different default value that is more appropriate for the project.